src/cmd/builtin/RELEASE

```diff
@@ -1,156 +0,0 @@
-16-01-06 added dummy code in builtin.c to keep the linker from optimizing it out
```

Comment:
We lost a bit of history here. This would be useful if we choose to keep `pty` command, but can look in legacy branches as an alternative.

Reply:
That was my thought. The history of this command isn't particularly useful so in the interest of eliminating useless cruft I just tossed the file. I'm hoping to convince people to drop the `pty` command so this really is focused on `ksh` and nothing else. Googling for mentions of this project's `pty` command I couldn't find any unambiguous mentions. The closest was these [course notes](https://www.ict.griffith.edu.au/anthony/info/co-processing/general_hints.txt) and it is far from obvious it is referring to the AST implementation.